assets/price-per-item.js

```diff
@@ -0,0 +1,103 @@
+if (!customElements.get('price-per-item')) {
+  customElements.define(
+    'price-per-item',
+    class PricePerItem extends HTMLElement {
+      constructor() {
+        super();
+        this.input = document.getElementById(`Quantity-${this.dataset.sectionId}`);
+        this.variantId = document.getElementById(`Price-Per-Item-${this.dataset.sectionId}`).dataset.variantId;
+        this.input.addEventListener('change', this.onInputChange.bind(this));
+
+        this.getVolumePricingArray();
+      }
+
+      updatePricePerItemUnsubscriber = undefined;
+      cartUpdateStartedUnsubscriber = undefined;
+      variantIdChangedUnsubscriber = undefined;
+
+      connectedCallback() {
+        // Update varinatId if variant is switched on product page
+        this.variantIdChangedUnsubscriber = subscribe(PUB_SUB_EVENTS.variantChange, (event) => {
+          this.variantId = event.data.variant.id.toString();
+          this.getVolumePricingArray();
+        });
+        this.cartUpdateStartedUnsubscriber = subscribe(PUB_SUB_EVENTS.cartUpdateStarted, (event) => {
+          if (event.variantId !== this.variantId) return;
+          this.querySelector('.loading-overlay').classList.remove('hidden');
+        });
+
+        this.updatePricePerItemUnsubscriber = subscribe(PUB_SUB_EVENTS.cartUpdate, (response) => {
+          this.querySelector('.loading-overlay').classList.add('hidden');
+          // Item was added to cart via product page
```

Comment:
Do we need these comments? 👀

Reply:
I find them helpful at least now when we are going to work with this code for featured product for example.